README.md

```diff
@@ -1,3 +1,91 @@
-## My test task
+# OrderBook
+
+---
 
 Here you can find my implementation of OrderBook.
+
+## Background:
```

Comment:
Add new lines to the 'Background' section.
![image](https://github.com/Tim2313/OrderBook/assets/27917560/45aaeb2f-4b10-4d19-bcf9-c9f1d970b414)

Reply:
Please, fix it

Reply:
fix

Reply:
There is still no new lines. Please add them, because text is unreadable.

Reply:
fix